setup.py

```diff
@@ -101,6 +101,47 @@ def parse_arg_remove_string(argv, arg_name_equal):
 is_manylinux = environ.get('AUDITWHEEL_PLAT', None) in manylinux_tags
 
 
+def build_torch_cpp_extensions():
+    # Run this from build dir (e.g. (...)/build/Linux/RelWithDebInfo/)
+    is_gpu_available = False
+    try:
+        import torch
+        is_gpu_available = torch.cuda.is_available()
+    except ImportError:
+        ...
+
+    ############################################################################
+    # Pytorch CPP Extensions that DO require CUDA/ROCM
+    ############################################################################
+    if is_gpu_available:
+        ret_code = subprocess.call(
+            f"python torch_cpp_extensions/torch_gpu_allocator/setup.py build {'--use_rocm' if rocm_version else ''}", shell=True
```

Comment:
Do we have to deal with cases where the Python interpreter isn't called `python`? (e.g. Is there some way to get the name of the current interpreter binary and use that as part of the call?)

Reply:
Yes, sys.executable would be better than hardcoded python